src/views/PopupView.tsx

```diff
@@ -34,17 +33,74 @@ export default function PopupView(
+                            coordinate system, but vector coordinates use OpenSim's</li>
+                    </ul>
+                    Timeline Track Area
+                    <ul>
+                        <li>The playback rate may be adjusted below the play button (after left clicking the current
+                            number) by either:
+                            <ul>
+                                <li>Typing any positive decimal value, such as 0.02</li>
+                                <li>Typing the up/down arrow keys to increment/decrement the number by 0.1</li>
+                            </ul>
+                        </li>
+                        <li>Suggested gait events should appear below the timeline track upon marker file upload</li>
+                        <li>To crop the animation start, left click any gait event button</li>
+                        <li>To crop the animation end, right click any gait event button</li>
+                        <li>Fine adjustments to the resulting crop may be made in the timeline text area</li>
+                        <li>To reset the current crop, right click anywhere in the timeline track</li>
+                    </ul>
+                    Timeline Text Area
+                    <ul>
+                        <li>The top row numbers are the frame numbers</li>
+                        <li>The bottom row numbers are the corresponding time in seconds</li>
+                        <li>Start refers to the cropped beginning frame of the data/animation (initially frame 0)</li>
+                        <li>End refers to the cropped ending frame of the data/animation (initially the last frame of the
+                            uploaded marker data)</li>
+                        <li>It is recommended to crop the end before cropping the start. Upon typing, cropping is applied
+                            immediately, so an interim end crop of frame 2 while typing 225 will be rejected if the crop
+                            start is greater than or equal to frame 2 to ensure there is always at least one frame to
+                            animate. Setting the end crop first avoids this issue. If needed, you can always right click
+                            the timeline track to reset the crop and start over.</li>
+                    </ul>
+                </div>
             </div>
             <div id={"menu-dropdown-container"}>
                 <div id={"menu-title"}>Main Menu</div>
                 <dl id={"menu-options"}>
                     <dt>Controls Help</dt>
                     <dd onMouseOver={()=>setControlsHelpImgNum(0)}>camera</dd>
                     <dd onMouseOver={()=>setControlsHelpImgNum(1)}>selection</dd>
-                    <dd onMouseOver={()=>setControlsHelpImgNum(2)}>timeline</dd>
+                    <dd onMouseOver={()=>setControlsHelpImgNum(2)}>more</dd>
```

Comment:
The image number for this will have to be 3 now too

Reply:
i believe i resolved the issues

Reply:
Looks good. The only other thing that might be better would be to put the "more" option below the "timeline" option, i.e. swap lines 106 and 107.

Reply:
k i did it